wiki/10_Jetpack-Compose.md

```diff
@@ -1,4 +1,6 @@
-# Jetpack Compose support
+# Jetpack Compose support [Early access]
+
+**Be aware of it's an early access that may contain bugs. Also, API can be changed, but we are going to avoid it. Please, be free to create relative issues if you've encountered with any kind of problem.**

```

Comment:
```suggestion
**Be aware of its early access that may contain bugs. Also, API can be changed, but we are going to avoid it. Be free to create relative issues if you've encountered any kind of problem.**

```